docker-compose.yml

```diff
@@ -377,10 +365,43 @@ services:
   # Grafana
   grafana:
     image: grafana/grafana:9.1.0
-    container_name: grafana
     volumes:
       - ./src/grafana/grafana.ini:/etc/grafana/grafana.ini
       - ./src/grafana/provisioning/:/etc/grafana/provisioning/
     ports:
       - "${GRAFANA_SERVICE_PORT}:${GRAFANA_SERVICE_PORT}"
     logging: *logging
+
+  # Tracetest
+  tracetest:
```

Comment:
Here's everything that is needed to start using Tracetest in local development.
1. Tracetest.
2. Progress.